SU2_CFD/include/variables/CVariable.hpp

```diff
@@ -105,6 +106,10 @@ class CVariable {
   unsigned long nSecondaryVar = 0;     /*!< \brief Number of secondary variables. */
   unsigned long nSecondaryVarGrad = 0;   /*!< \brief Number of secondaries for which a gradient is computed. */
 
+  /*!< \brief Inddex definition for NEMO variables. */
+  unsigned long RHOS_INDEX, T_INDEX, TVE_INDEX, VEL_INDEX, P_INDEX, 
+  RHO_INDEX, H_INDEX, A_INDEX, RHOCVTR_INDEX, RHOCVVE_INDEX, nSpecies;
+
```

Comment:
Take this out of CVariable please, it does not make sense to have it here visible to other variable classes that have nothing to do with NEMO or even flow problems.